proto/proto-backplane-grpc/src/main/proto/deephaven/proto/partitionedtable.proto

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+ */
+
+syntax = "proto3";
+
+package io.deephaven.proto.backplane.grpc;
+
+option java_multiple_files = true;
+option optimize_for = SPEED;
+
+import "deephaven/proto/table.proto";
+import "deephaven/proto/ticket.proto";
+
+/*
+ * This service provides tools to create and query partitioned tables.
+ */
+service PartitionedTableService {
+  /*
+   * Transforms a table into a partitioned table, consisting of many separate table, each individually
+   * addressable. The result will be a FetchObjectResponse populated with a PartitionedTable.
+   */
+  rpc PartitionBy(PartitionByRequest) returns (PartitionByResponse);
+  /*
+   * Given a partitioned table, returns a table with the contents of all of the constituent tables.
+   */
+  rpc Merge(MergeRequest) returns (ExportedTableCreationResponse);
+  /*
+   * Given a partitioned table and a row indicated by position or by the values of a 1-row table,
```

Comment:
Yeah I removed that ability - do you think it is worth restoring, so that a sufficiently smart client could query by row key/position at all? Or just assume we want safer operations?